src/main/java/com/google/codeu/data/Message.java

```diff
@@ -101,7 +121,7 @@ public void setImageUrl(String newImageUrl) {
   }
 
   public boolean hasAnImage() {
-    if (this.imageUrl.equals("")) {
+    if (this.imageUrl == null || this.imageUrl.isEmpty()) {
```

Comment:
Previous version threw up errors on messages which had been created before the image functionality. This seemed to fix it